app/javascript/onboarding/components/FollowTags.jsx

```diff
@@ -74,30 +74,45 @@ class FollowTags extends Component {
           credentials: 'same-origin',
         }),
       ),
-    ).then(_ => {
+    ).then((_) => {
       const { next } = this.props;
       next();
     });
   }
 
+  renderFollowCount() {
+    const { selectedTags } = this.state;
+    let followingStatus;
+    if (selectedTags.length < 2) {
+      followingStatus = `${selectedTags.length} topic selected`;
```

Comment:
What if `selectedTags.length` is `0`? Based on [the snapshot](https://github.com/thepracticaldev/dev.to/pull/7293/files#diff-9560b1a73a8f29f53129b0b6cadf5fd9R270), it looks like it won't pluralize correctly.

Reply:
Nice catch! Updated